### PR TITLE
refactor: Add the option to force a build on staging

### PR DESCRIPTION
**Pull request by jorelosorio**
This adds a new input var to force a build in the staging action.

### CODE REVIEW

File: .github/workflows/build-staging-ios.yml [hunk 1]
```diff
@@ -60,7 +69,7 @@ jobs:
           GH_ACCESS_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
           MATCH_PASSWORD: ${{ secrets.MATCH_PASSWORD }}
       - name: Replace ipa bundle
-        if: steps.ipa-cache.outputs.cache-hit == 'true'
+        if: inputs.force-build == 'true' || steps.ipa-cache.outputs.cache-hit == 'true'
```
Comment (tormoseng) [inline .github/workflows/build-staging-ios.yml:72]: The other changes are logical to me, but what does `inputs.force-build == 'true' || ...` implies here?

Comment (jorelosorio) [reply .github/workflows/build-staging-ios.yml]: It should run it in any case but then I looked at another build, for example, the cache was changed and this is skipped, so I changed the logic a bit to fit.

**Review by tormoseng — APPROVED** [review]
LGTM!

**Comment by tormoseng** [comment]
Merged on behalf of @jorelosorio

**Pull request merged**